masonry/src/widget/widget_pod.rs

```diff
@@ -337,12 +337,10 @@ impl<W: Widget> WidgetPod<W> {
 
                 true
             }
-            LifeCycle::AnimFrame(_) => {
-                state.request_anim = false;
-                true
-            }
             // Routing DisabledChanged has been moved to the update_disabled pass
             LifeCycle::DisabledChanged(_) => false,
+            // Animations have been moved to the update_anim pass
+            LifeCycle::AnimFrame(_) => false,
```

Comment:
Is this `unreachable!`?

Reply:
Not quite. A container widget could request anim. If it recurses the lifecycle function over children, then that line of code is reached.